### ISSUE TITLE
JSDoc.getInnerText() strips leading spaces from inside the jsdoc

### ISSUE THREAD

**Issue by gregjacobs** [dsherret/ts-simple-ast#239]
Hey David,

Just noticed something: the `JSDoc.getInnerText()` method strips leading spaces in JSDoc comments, which are used to denote code examples. Code examples are indented by 4 spaces, which is treated as a code block by the documentation generator. For example:

```ts
/**
 * A description of my method.
 *
 * Example usage:
 *
 *    myMethod().then(doSomething);
 */
```

`getInnerText()` returns this:

```
A description of my method

Example usage:

myMethod().then(doSomething);
```

When it should return this:

```
A description of my method

Example usage:

    myMethod().then(doSomething);
```

**Comment by dsherret** [comment]
True, thanks! I'll have it fixed soon.

**Comment by dsherret** [comment]
@gregjacobs this will be in the next release (I'll probably do one tonight). Thanks for reporting this!

**Comment by dsherret** [comment]
Ok, this is available in 6.9.0

**Comment by gregjacobs** [comment]
Awesome, thanks man!
